src/content/whats-new/2022/03/whats-new-03-01-Kubernetes-APM.md

```diff
@@ -0,0 +1,24 @@
+---
+title: 'Minimize Kubernetes complexity to build performant applications'
+summary: 'Analyze application and Kubernetes observability data in one place'
+releaseDate: '2022-03-01'
+learnMoreLink: 'https://newrelic.com/blog/nerdlog/kubernetes-experience-in-APM'
+---
+## See Kubernetes details in APM
+
+Use our new UI that shows how cluster performance impacts your applications.
+
+* Analyze cluster performance in a single, curated UI.
+* Get alerted with a real-time activity stream for Kubernetes events and critical issues.
+* Correlate performance anomalies with one-click logs and side-by-side (selectable) metrics.
+
+## How to get started
+Kubernetes information in APM is available in **[one.newrelic.com](http://one.newrelic.com) > Explorer > Services - APM > (select an app) > Kubernetes**.
+
+To make sure your Kubernetes performance data appears in New Relic One:
+
+* Install the New Relic [infrastructure integration](https://docs.newrelic.com/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm/).
+* See our general [APM](https://docs.newrelic.com/docs/apm/new-relic-apm/getting-started/introduction-apm/) documentation, as well as information about the [Kubernetes summary page](https://docs.newrelic.com/docs/apm/apm-ui-pages/monitoring/kubernetes-summary-page).
```

Comment:
The Kubernetes Summary Page linked in docs is giving an error message.

Reply:
@alexiskjones, the link to the Kubernetes page in APM will fail in this PR because it hasn't been merged to the develop  branch that has this new document. I was just told that we need to pull back the entire feature, including the "What's new" post.

Reply:
Thanks. Yes, we will hold on pushing this What's New post until 3/2 (we're in a holding pattern until the Logs team can work out an issue with a feature). The APM docs page can go live as planned.